src/deployment/android.md

```diff
@@ -259,6 +259,15 @@ and will ask before making changes to your Android project. Opting in allows
 Flutter to automatically depend on `androidx.multidex:multidex` and use a
 generated `FlutterMultiDexApplication` as the project's application.
 
+When you try to build and run your app with the **Run** and **Debug** options in your IDE, your build may fail with the following message:
+<img src='/assets/images/docs/deployment/android/ide-build-failure-multidex.png' width="100%" alt='screenshot of build failure because Multidex support is required'>
```

Comment:
Should this be `<img src='/assets/images/docs/android/ide-build-failure-multidex.png'` instead?

Alternatively, move the image assets to match the above code?

Reply:
I'll move the images to keep the paths of the links in the file consistent.